### DISCUSSION TITLE
Escape bracket character appears in the docs

### DISCUSSION THREAD

**Discussion by HarelM** [squidfunk/mkdocs-material#6682] [Q&A] [answered]
### Context

I'm trying to show an API with "generic"/"template" code such as `Promise<T>` but the escape character is presented.
Can be seen here:
![image](https://github.com/squidfunk/mkdocs-material/assets/3269297/22e2cfa5-25e5-41c2-92da-617167bb15c2)

### Bug description

I'm not sure what's causing this, and if this is a markdown specification that is different from Github for example.

### Related links

https://maplibre.org/maplibre-gl-js/docs/API/ and search for "once".

### Reproduction

See the below, I hope it will be enough, I'm not doing anything special I believe, my config can be found here:
https://github.com/maplibre/maplibre-gl-js/blob/9d5b1b4afe1cbb64dd6215470de4a6900eca76d1/mkdocs.yml

### Steps to reproduce

The following is the markdown:
```
### once

▸ **once**\<`T`\>(`type`, `layer`, `listener?`): [`Map`](Map.md) \| `Promise`\<[`MapLayerEventType`](../types/maplibregl.MapLayerEventType.md)[`T`] & `Object`\>
```

### Before submitting

- [X] I have read and followed the [bug reporting guidelines](https://squidfunk.github.io/mkdocs-material/contributing/reporting-a-bug/).
- [X] I have attached links to [the documentation](https://squidfunk.github.io/mkdocs-material/), and possibly related [issues](https://github.com/squidfunk/mkdocs-material/issues) and [discussions](https://github.com/squidfunk/mkdocs-material/discussions).
- [X] I assure that I have [removed all customizations](https://squidfunk.github.io/mkdocs-material/contributing/reporting-a-bug/#remove-customizations) before submitting this bug report.
- [X] I have attached a __.zip file__ with a [minimal reproduction](https://squidfunk.github.io/mkdocs-material/guides/creating-a-reproduction/).

**Comment by facelessuser** [comment, marked as answer]
Maybe try using [EscapeAll](https://facelessuser.github.io/pymdown-extensions/extensions/escapeall/).

<img width="760" alt="Screenshot 2024-01-22 at 3 15 07 PM" src="https://github.com/squidfunk/mkdocs-material/assets/1055125/d6f61528-2c36-46ff-a698-0d4e754f7e3c">

**Reply by HarelM** [reply]
Is this a plugin? How can I use it in mkdocs?

**Reply by HarelM** [reply]
Found it, thanks!! This is looking great now!

**Reply by ShellyDCMS** [reply]
for anyone else coming across this issue, add escapeall to your mkdocs yml file:
![image](https://github.com/user-attachments/assets/1aae3152-4bae-44b4-b91f-c08429b38dfa)

**Comment by squidfunk** [comment]
Thanks for reporting. Converting to discussion, since it is not something that is particular to Material for MkDocs, but to all MkDocs themes. Next time, also please stick to our bug reporting process and provide a minimal reproduction.
